yarn-project/end-to-end/src/e2e_p2p/slashing.test.ts

```diff
@@ -215,8 +239,15 @@ describe('e2e_p2p_slashing', () => {
       sInfo = await slashingInfo(roundNumber);
       t.logger.info(`We have ${sInfo.leaderVotes} votes in round ${sInfo.roundNumber} on ${sInfo.info[1]}`);
       if (sInfo.leaderVotes >= votesNeeded) {
-        t.logger.info(`We have sufficient votes`);
-        break;
+        // We need there to be an actual committee to slash for this round
+        const epoch = await rollup.getEpochNumberForSlotNumber(sInfo.slotNumber);
+        const committee = await rollup.getEpochCommittee(epoch);
+        if (committee.length > 0) {
+          t.logger.info(`We have sufficient votes, and a committee for epoch ${epoch}`);
+          break;
+        } else {
+          t.logger.info(`No committee found for epoch ${epoch}, waiting for next round`);
```

Comment:
How much more slow do we end making the setup by this? Just thinking that we could probably benefit earlier in the test from ensuring that there are people in there. A potential speedup could be to not alter the slashing amount to be non-zero until after we are sure there are committee members. Or guess it won't necessarily work that well if the offence was made earlier already.

Reply:
Since we make the timejump at the beginning, it should add no time at all. If no time jump is made, this check means the test will still work